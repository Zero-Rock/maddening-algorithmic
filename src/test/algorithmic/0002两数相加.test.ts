// tslint:disable:no-magic-numbers
import { addTwoNumbers, ListNode } from 'src/algorithmic/0002.两数相加';
import { describe, expect, test } from 'vitest';

const l11 = new ListNode(6, null);
const l12 = new ListNode(5, l11);
const l13 = new ListNode(1, l12);

describe('两数相加', () => {
  test('should be worked well', () => {
    expect(addTwoNumbers(l13, l13)).toMatchSnapshot();
  });
});
