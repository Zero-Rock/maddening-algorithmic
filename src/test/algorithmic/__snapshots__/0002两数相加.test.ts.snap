// Vitest Snapshot v1

exports[`两数相加 > should be worked well 1`] = `
ListNode {
  "next": ListNode {
    "next": ListNode {
      "next": null,
      "val": 2,
    },
    "val": 2,
  },
  "val": 2,
}
`;
